packages/flutter/analysis_options.yaml

```diff
@@ -5,3 +5,7 @@ include: ../analysis_options.yaml
 analyzer:
   enable-experiment:
     - non-nullable
+  errors:
+    always_require_non_null_named_parameters: false # not needed with nnbd
+    void_checks: false # issues with nnbd
```

Comment:
Is there an issue on file that we could link from here?

Reply:
link to issue added.